frontend/src/components/About.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import { H2, P } from './UI/Typography';
+import Logo from './logo.jpg';
+
+const Wrapper = styled.div`
+display: flex;
+justify-content: center;
+align-items: center;
+`;
+
+const H1 = styled(H2)`
+    font-size: 50px;
+    text-align:center;
+    color: ${(props) => props.theme.colors.primary};
+    padding: ${(props) => props.theme.padding.section};
+`;
+
+const H3 = styled(H2)`
+    font-size: 38px;
+    font-family: Kufam, sans-serif;
+    text-align: left;
+`;
+
+const P1 = styled(P)`
+    font-size: 18px;
+    font-family: Kufam, sans-serif;
+    text-align: left;
+`;
```

Comment:
Rekommenderar att du tar bort font-sizes från dessa. Det är okej att öka storleken på text utifrån det som finns definierat i Typography.jsx men du har redan gjort det för din H1. Om man ökar storleken på all text så blir det en ny standard och skärmen passar inte in bland våra andra komponenter.

`font-family: Kufam, sans-serif;` behövs inte eftersom Typography.jsx och dess header redan har en sådan property.